python-avd/pyavd/_eos_designs/structured_config/underlay/loopback_interfaces.py

```diff
@@ -70,14 +82,24 @@ def loopback_interfaces(self: AvdStructuredConfigUnderlay) -> list | None:
             and self.shared_utils.vtep_loopback.lower() != "loopback0"
             and self.shared_utils.vtep_loopback.lower().startswith("lo")
         ):
-            vtep_loopback = {
-                "name": self.shared_utils.vtep_loopback,
-                "description": self.shared_utils.interface_descriptions.vtep_loopback_interface(
-                    InterfaceDescriptionData(shared_utils=self.shared_utils, interface=self.shared_utils.vtep_loopback)
-                ),
-                "shutdown": False,
-                "ip_address": f"{self.shared_utils.vtep_ip}/32",
-            }
+            if not self.shared_utils.underlay_ipv4:
```

Comment:
same as above, please only if/else on the ip address part since the rest is the same.